### ISSUE TITLE
Timeline: Linked timelines fires too many ajax requests

### ISSUE THREAD

**Issue by jasonex7** [primefaces/primefaces#5254]
## 1) Environment
- PrimeFaces version: 7.0 and 7.1-SNAPSHOT
- Does it work on the newest released PrimeFaces version? No
- Does it work on the newest sources in GitHub? No

## 2) Expected behavior

In [Linked Timelines example](https://www.primefaces.org/showcase/ui/data/timeline/linked.xhtml) when you move the first timeline, then only one ajax request should be fired.

## 3) Actual behavior

In [Linked Timelines example](https://www.primefaces.org/showcase/ui/data/timeline/linked.xhtml) when you move the first timeline, then too many ajax requests fired.

## 4) Steps to reproduce
![d7781e96-510c-4adb-88fd-7b63f580aac4](https://user-images.githubusercontent.com/16229253/67103142-3ba59900-f18a-11e9-965d-7c90df9e28ab.gif)

#### Reference

> if you open the showcase (http://localhost:8080/showcase/ui/data/timeline/linked.xhtml)
> and zoom or drag something, a endless loop of AJAX requests will be fired
> can you please fix that?

_Originally posted by @tandraschko in https://github.com/primefaces/primefaces/pull/5113#issuecomment-543603744_

**Comment by melloware** [comment]
And Vis Timeline 6.0 was just released too: https://github.com/visjs/vis-timeline/releases/tag/v6.0.0

**Comment by jasonex7** [comment]
> And Vis Timeline 6.0 was just released too: https://github.com/visjs/vis-timeline/releases/tag/v6.0.0

Excellent news! Do you recommend open another issue for updating to v6.0.0?

**Comment by melloware** [comment]
No I think you can make it part of this PR if you fix this Ajax issue.

**Comment by melloware** [comment]
Looks like they did a patch release to 6.0.3: https://github.com/visjs/vis-timeline/releases/tag/v6.0.3

**Comment by jasonex7** [comment]
I've investigated how was the execution of the original example in showcase of PrimeFaces Extensions and saw that no ajax request was fired on moving timeline, only when user selected an item. See the following capture:

![7df0e5fc-1a2b-4132-82b0-f53f648fe1cb](https://user-images.githubusercontent.com/16229253/67215168-9dfad580-f3e6-11e9-8d53-d9b4eb0bd061.gif)

So the main difference is that the current showcase example uses `<p:ajax>` tag with `oncomplete` property and the old extension example uses `<pe:javascript>` tag with `execute` property which doesn't fire any ajax request and executes only client side code.

_Current Showcase Example_
	![image](https://user-images.githubusercontent.com/16229253/67217068-ccc67b00-f3e9-11e9-8ea7-4da03305ecc3.png)

_Extensions Example_
	![image](https://user-images.githubusercontent.com/16229253/67216360-b1a73b80-f3e8-11e9-9fba-dc029deb7c74.png)

So, in other words, this is a showcase issue and not a PrimeFaces Library Issue. In order to solve this issue we would have to modify the example to not fire ajax request on rangechange event and only execute the client side code.

Another thing to mentions is that "linked timeline example" was in the old Timeline based on CHAP Links Library only (see [link](https://almende.github.io/chap-links-library/js/timeline/examples/example10_link_two_timelines.html)), but then it disappear from vis based Timeline examples, but the good news is I've found a way to make it functional with current vis-timeline:
```xhtml
    <script type="text/javascript">
        //<![CDATA[
        function onrangechange1() {
            var second = PF('timelineSecondWdgt');
            if (second.jq.data("rangeFired")) {
                second.jq.data("rangeFired", null);
                return;
            }
            var first = PF('timelineFirstWdgt');
            var range = first.getVisibleRange();
            first.jq.data("rangeFired", true);
            PF('timelineSecondWdgt').setVisibleRange(range.start, range.end);                
        }

        function onrangechange2() {
            var first = PF('timelineFirstWdgt');
            if (first.jq.data("rangeFired")) {
                first.jq.data("rangeFired", null);
                return;
            }
            var second = PF('timelineSecondWdgt');
            var range = second.getVisibleRange();
            second.jq.data("rangeFired", true);
            PF('timelineFirstWdgt').setVisibleRange(range.start, range.end);                
        }
        //]]>
    </script>

    <h3 style="margin-top: 0">Events</h3>
    <p:timeline id="timelineFirst" value="#{linkedTimelinesView.modelFirst}" var="task"
                height="250px" widgetVar="timelineFirstWdgt">
        <p:ajax event="rangechange" process="@none" onstart="onrangechange1(); return false;"/>
        <p:ajax event="select" listener="#{linkedTimelinesView.onSelect}"/>

        <h:panelGroup layout="block" rendered="#{not task.period}" style="padding-bottom:5px">
            <h:outputText value="#{task.title}"/>
        </h:panelGroup>
        <p:graphicImage library="demo" name="#{task.imagePath}"/>
        <h:panelGroup rendered="#{task.period}" style="padding:8px">
            <h:outputText value="#{task.title}"/>
        </h:panelGroup>
    </p:timeline>

    <h3>Projects</h3>
    <p:timeline id="timelineSecond" value="#{linkedTimelinesView.modelSecond}"
                height="150px" widgetVar="timelineSecondWdgt">
        <p:ajax event="rangechange" process="@none" onstart="onrangechange2(); return false;"/>
    </p:timeline>
```
This should be resolved in primefaces/showcase-facelift#63
